### DISCUSSION TITLE
Unable to start Peacock

### DISCUSSION THREAD

**Discussion by shin-chan-cpu** [idaholab/moose#27969] [Q&A Getting Started] [answered]
### Check these boxes if you have followed the posting rules.

- [X] Q&A Getting Started is the most appropriate category for my question (trouble installing, beginner user, ...)
- [X] I have consulted the posting Guidelines on the Discussions front page
- [X] I have searched the Discussions forum and my question has not been asked before
- [X] I have searched the MOOSE website and the documentation does not answer my question
- [X] I have formatted my post following the posting guidelines (avoid screenshots if possible, triple back quotes before/after pasted text, etc)

### Issue or question about MOOSE

I installed Peacock under Ubuntu, and I added export PATH =  "/home/wangyaofeng/projects/moose/python/peacock: $PATH" to~/.bashrc,~/bash_profile and~/.profile .This command,peacock is also clearly in this directory, but the feedback from the end point is still peacock: command not found, I don't know where the problem occurs?

### Diagnostics for MOOSE installation issues

[Optional] Output of the diagnostics scripts. To run the script: cd ~/projects/moose/scripts; ./diagnostics.sh

**Comment by GiudGiud** [comment]
Hello

what does "echo $PATH" report?

Guillaume

**Reply by shin-chan-cpu** [reply]
it report these /home/wangyaofeng/miniforge/envs/peacock/bin:/home/wangyaofeng/miniforge/condabin:/usr/local/sbin:/usr/local/bin:/usr/sbin:/usr/bin:/sbin:/bin:/usr/games:/usr/local/games:/snap/bin:/snap/bin

**Reply by GiudGiud** [reply]
seems it did not work then. It should have `/home/wangyaofeng/projects/moose/python/peacock` in there

so try:
```
export PATH=/home/wangyaofeng/projects/moose/python/peacock:$PATH
```

**Reply by shin-chan-cpu** [reply]
It still doesn't work. Does this have a lot to do with my. bash_profile file being empty? I created this file

**Reply by GiudGiud** [reply]
which shell are you using?
what does `echo $0` return?

is the PATH as expected now?

**Reply by shin-chan-cpu** [reply]
it returns bash,but it doesn't exist .bash_profile, I just build it and expexted PATH

**Reply by milljm** [reply, marked as answer]
explicitly, what does the following report:

```bash
cat ~/.bash_profile
```

Also if you _literally_ entered what you say you entered in your original post, it will not work (all these spaces need to go away):
```bash
PATH = "/home/wangyaofeng/projects/moose/python/peacock: $PATH" 
```
_needs to be_:
```bash
export PATH="/home/wangyaofeng/projects/moose/python/peacock:$PATH" 
```
